awx_collection/plugins/modules/job_template.py

```diff
@@ -320,8 +320,8 @@
 
 notes:
   - JSON for survey_spec can be found in the API Documentation. See
-    U(https://docs.ansible.com/ansible-tower/latest/html/towerapi/api_ref.html#/Job_Templates/Job_Templates_job_templates_survey_spec_create)
-    for POST operation payload example.
+    U(https://developers.redhat.com/api-catalog/api/ansible-automation-controller) and search for 'survey' for information on job template survey creation
+    and POST operation payload example.
```

Comment:
I'd probably point to https://ansible.readthedocs.io/projects/awx/en/latest/rest_api/api_ref.html#/Job_Templates/Job_Templates_job_templates_survey_spec_list

Reply:
It's hard to tell, @samccann. I'll need to discuss further with @gundalow about this one.

Reply:
After some discussion with @gundalow, I'll point readers to a [landing page](https://docs.ansible.com/automation-controller/latest/html/towerapi/) that has links for them to go to, depending on their situation. Since this page will ultimately go away, we will keep this an open issue (tracked elsewhere) until we are directed to do something different.